modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc

```diff
@@ -2,57 +2,95 @@
 
 Start a local container cluster.
 
+This command utilizes Docker to initiate a local container cluster. Use the `--nodes` (or the shorthand version `-n`) flag to specify the number of brokers.
+
+The initial broker starts on default ports, with subsequent brokers' ports offset by 1000. You can use the following flags to specify listeners ports:
+
+* --kafka-ports
+
+* --admin-ports
+
+* --rpc-ports
+
+* --schema-registry-ports
+
+* --proxy-ports
+
+Each flag accepts a comma-separated list of ports for your listeners. Use the `--any-port` flag to let `rpk` randomly select an available port on the host machine.
+
+In case of IP address pool conflict, you may specify a custom subnet and gateway using the `--subnet' and `--gateway` flags respectively.
```

Comment:
```suggestion
In case of IP address pool conflict, you may specify a custom subnet and gateway using the `--subnet` and `--gateway` flags respectively.
```